patches/api/0156-Add-Material-Tags.patch

```diff
@@ -582,6 +584,16 @@ index 0000000000000000000000000000000000000000..f5ae132cef01666bacda11422c0b73dd
 +        .ensureSize("HOES", 6).lock();
 +
 +    /**
++     * Covers all variants of armor.
++     */
++    public static final MaterialSetTag ARMOR = new MaterialSetTag(keyFor("armor"))
++        .add(material -> material.name().endsWith("_HELMET") ||
++            material.name().endsWith("_CHESTPLATE") ||
++            material.name().endsWith("_LEGGINGS") ||
++            material.name().endsWith("_BOOTS"))
```

Comment:
You can add tags to tags. So just add the HELMET, CHESTPLATE, LEGGINGS, and BOOTS tag to this one tag. No need to do the whole endsWith thing again as thats already done below for those other 4 tags

Reply:
Fixed 👍